src/runtime/contributions-flow.js

```diff
@@ -73,6 +76,7 @@ export class ContributionsFlow {
                 'skus': (options && options.skus) || null,
                 'isClosable': isClosable,
                 'supportsEventManager': true,
+                'shouldDisableBodyScrolling': shouldDisableBodyScrolling,
```

Comment:
I think these are args that are passed into the iframe to receive. Any reason this prob can't just be passed into as a `DialogConfig` setting? https://github.com/subscriptions-project/swg-js/blob/f56b586bd4cb92c443b2db1667e1cc677123d8c6/src/components/dialog.js#L69-L85
I imagine it'd reduce the amount of filed being touched for that change

Reply:
Updated. Thanks for the suggestion!